KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutViewController.swift

```diff
@@ -428,7 +430,14 @@ final class KarhooCheckoutViewController: UIViewController, CheckoutView {
         backButton.isUserInteractionEnabled = false
         backButton.tintColor = KarhooUI.colors.medGrey
     }
-    
+
+    var areTermsAndConditionsAccepted: Bool { termsConditionsView.isAccepted }
```

Comment:
Could we organize the methids in this class a bit? Everything is under the Checkout View Methods mark. Maybe by doing this we could also move this var up with the rest of them?